habitat-baselines/habitat_baselines/rl/ppo/habitat_evaluator.py

```diff
@@ -302,6 +302,10 @@ def evaluate_agent(
                 rgb_frames,
             )
 
+            # We pause the statefull parameters in the policy
+            if any(envs_to_pause):
```

Comment:
Why add this? Should `on_envs_pause` should be able to handle empty lists of ends to pause?